src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -673,4 +593,16 @@ public static String boldHTML(String input, String alternative) {
         return "<b>" + input + "</b>";
     }
 
+    /**
+     * Quotes each and every character, e.g. '!' as &#33;. Used for verbatim
+     * display of arbitrary strings that may contain HTML entities.
+     */
+    public static String quoteForHTML(String toQuote) {
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < toQuote.length(); ++i) {
+            result.append("&#").append((int) toQuote.charAt(i)).append(';');
```

Comment:
No, because then you get `.append(char)` which will not output a number. `Integer.valueOf(char)` may be a more clear alternative?

 (I should have moved it back to the original place ;-)).